packages/google_maps_flutter/google_maps_flutter_web/example/integration_test/google_maps_plugin_test.dart

```diff
@@ -74,16 +78,7 @@ void main() {
 
       testWidgets('throws without _webOnlyMapCreationId',
           (WidgetTester tester) async {
-        expect(
-          () => plugin.buildView(
-            null,
-            onPlatformViewCreated,
-            initialCameraPosition: initialCameraPosition,
-          ),
-          throwsAssertionError,
-          reason:
-              '_webOnlyMapCreationId is mandatory to prevent unnecessary reloads in web.',
-        );
+        // The argument type 'Null' can't be assigned to the parameter type 'int'.
       });
```

Comment:
In fact, I'm going to try and create a couple of small @dart=2.9 test with these tests, see what happens :P

Reply:
In the end, since this new package is a major version, I've decided to drop support for this feature, and remove the obsolete tests. I've added a "breaking changes" section to the README to document these changes in behavior. Thanks for bringing them up!